content/articles/building-a-lyrics-search-app-using-vanilla-javascript/index.md

```diff
@@ -0,0 +1,490 @@
+            <h1>Learn your favourite</h1>
+            <h2>song lyrics</h2>
+        </div>
+
+        <div id="lyrics-serach">
+            <form action="" id="searchMe">
+                <input type="text" id="lyricSearch" placeholder="artist name or song title"/>
+            </form>
+            <div id="search-result">
+                
+            </div>
+        </div>
+    </div>
+    <script src="lyrics.js">
+        
+    </script>
+</body>
+</html>
+
+```
+
+### Designing the lyrics search app
+The next step is to design the web app with CSS by navigating to your CSS file. Here’s the CSS for the structure (HTML code) of the web app:
+
+First, style the overall HTML, body of the HTML, and the div with the class `container` which embodies all other divs.
+
+```css
+html {
+    box-sizing:border-box;
+}
+
+/**Use pseudo elements to set some static styles**/
+*,
+*:before,
+*:after {
+      margin:0;
+      Padding:0;
+}
+
+body {
+      background:#333333;
+}
+
+.container {
+    margin:0 auto;
+}
+
+```
+
+Now, you are going to style the contents in the body tag, starting from the text that defines what the web app does, to the placeholder and then the input field.
+
+```css
+
+/**This is how placeholders are styled**/
+::placeholder {
+        color:gold;
+}
+
+```
+
+Next, style up the page description text (Learn your favorite song lyrics).
+
+```css
+
+/**Styling the intro text div which has the content of “learn your favorite song” lyrics**/
+
+.intro-text {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;/**If the device doesn’t have any of the first three font-family sans-serif will automatically be used **/
+      padding:2rem 0;
+     Color:gold;
+}
+
+/**Styling the h1 header(learn your favourite):centering the h1 element**/
+h1 {
+      text-align:center;
+      padding:1rem 0;
+}
+
+/**Styling the h1 header(learn your favourite):centering the h1 element**/
+h2 {
+      text-align:center;
+      padding:1rem 0;
+}
+ /**You might decide to use the comma-separated form of styling tags with the same properties. I decided to be more explicit here for the sake of clarity**/
+```
+
+Centering the div container where the form and input field are embedded, using its id (lyrics-search).
+
+```css
+#lyrics-search {
+    text-align:center;
+    align-items:center;
+    margin:20px;
+    padding:20px;
+}
+
+```
+
+You will complete the css code for now by styling up the form and the input field.
+
+```css
+form {
+    margin:0 auto;
+    padding-left:10rem;
+}
+
+/**style the input field using its id(lyricSearch)**/
+
+#lyricSearch {
+        background-color: transparent;
+        color:#eeeeee;
+        outline:none;
+        height:30px;
+        width:64%;
+        margin:0 auto;
+        padding-right:2rem;
+        font-size:16px;
+        font-family:cursive;
+        outline-style:none;
+        border-top:none;
+        border-left:none;
+        border-right:none;
+        border-bottom:1px solid #eeeeee;
+}
+```
```

Comment:
Usually, there is nothing much to explain in CSS unless the whole article is about CSS.

If you're not doing anything significant, I would suggest you to remove a few of them. Instead, add a reference link mentioning the GitHub repo containing the same contents.

Reply:
If you're adding some important attributes, make sure to add comments/explain them.